docs/videos/index.html

```diff
@@ -62,6 +62,16 @@ <h4>Short Introduction to Web of Things</a></h4>
 		</a>
         </div>
     </div>
+	<br /><br />
+    <div class="row">
+        <div class="col-md-6">
+		<a target="_blank" href="https://www.youtube.com/watch?v=bPxIfZo7jns">
+			<img class="img-responsive" style="min-width: 180px; width: 100%; border-width: thin; border-style: solid;" src="{{'/images/video-thumbnail-wam.png' | absolute_url }}" alt="WAM Video" />
+			<h4>Building a WoT application: WoT application Manager and TD-code</h4>
+			<!-- 2020-9-18 -->
```

Comment:
If the date is correct it should come first...

Reply:
yes I agree we should be fair and put more recent videos on top.